.github/workflows/tests.yml

```diff
@@ -343,7 +344,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: [3.8]
```

Comment:
For some reason, the prepare_env seems to fail with Python 3.9.

See https://github.com/SCALE-MS/scale-ms/actions/runs/3009098321

@andre-merzky any ideas?

It looks like we might not be getting all of the artifacts. Do the regular expressions need updating, or is it just the errors from files that are still open while tar is running?

Reply:
It looks like the testenv created in GitHub Actions might not be using the version from `${{ matrix.python-version }}`. I will investigate.

Reply:
I don't understand. The `env.log` reports
```
[/home/runner/testenv/bin/radical-pilot-create-static-ve -d -p /home/runner/radical.pilot.sandbox/rp.session.fv-az456-821.runner.019242.0007/pilot.0000//env/rp_named_env.scalems_env -t virtualenv -v 3.9 -m pip,setuptools,wheel,/home/runner/radical.pilot.sandbox/rp.session.fv-az456-821.runner.019242.0007/pilot.0000/radical.utils-1.16.0.tar.gz,/home/runner/radical.pilot.sandbox/rp.session.fv-az456-821.runner.019242.0007/pilot.0000/radical.saga-1.16.0.tar.gz,/home/runner/radical.pilot.sandbox/rp.session.fv-az456-821.runner.019242.0007/pilot.0000/radical.pilot-1.16.0.tar.gz,/home/runner/radical.pilot.sandbox/rp.session.fv-az456-821.runner.019242.0007/pilot.0000/scalems-0+untagged.1.g3afd2d8.tar.gz -P . env/bs0_pre_0.sh]
pre exec: . env/bs0_pre_0.sh
pre exec done
requested version 3.9 not supported for venv (3.8)
```
But `/home/runner/testenv/` is definitely a Py 3.9 venv, according to paths and `radical-stack`.